Scripts/Python/ki/xKIChat.py

```diff
@@ -866,27 +866,38 @@ def __call__(self, message):
                 else:
                     statusMsg = PtGetLocalizedString(emote[1], [PtGetLocalPlayer().getPlayerName()])
                 self.chatMgr.DisplayStatusMessage(statusMsg, 1)
+
+                # Get remaining message string after emote command
                 message = message[len(words[0]):]
-                if message == "":
+                if message == "" or message.isspace():
```

Comment:
Seeing as how this is repeated a few times, I wonder if it would make more sense to compile the regex `\s*` somewhere and use that as our magic "is this a blank or empty string" test.